docs/en/observability/aws-elastic-serverless-forwarder.asciidoc

```diff
@@ -0,0 +1,99 @@
+:aws: AWS
+
+[[aws-elastic-serverless-forwarder]]
+= Elastic Serverless Forwarder
+
+// should we expand titleabbrev to AWS or Amazon Web Services Elastic Serverless Forwarder? What is correct nomenclature?
+
+++++
+<titleabbrev>AWS Elastic Serverless Forwarder</titleabbrev>
+++++
```

Comment:
Colleen's suggestion is inline with their trademark guidelines.

Reply:
Nothing here that titleabbrev is also used for generated link text.

Reply:
> Or if the name of the product is "Elastic Serverless Forwarder" we could use something like "Elastic Serverless Forwarder for Amazon Web Services (AWS)".

We definitely need clarity on naming and how we connect it to AWS. I like this suggestion as it handles possible developments like Elastic Serverless Forwarder for BlahBlah (I know that other cloud platforms implement differently but it's not hard to imagine same concept being used for other cloud providers).

As we don't have the **Cloud observability** section yet, and the AWS bit is needed to make the context clear, I'd experimented with using titleabbrev to see how the menu item looks. I could see us having a longer page title and filenames but using the abbrev to shorten (e.g. AWS rather than Amazon Web Services). What do you think to the naming point @ravikesarwani?

Reply:
Any thoughts on the naming issue @ravikesarwani ?

Reply:
Using AWS in the titleabbrev is fine, so something like "Elastic Serverless Forwarder for AWS" should be okay from my perspective (trying to keep it smaller)